src/test/resources/samples/bibframe/authority_concept.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "123456654321",
+  "label": "concept-label",
+  "type": "CONCEPT",
+  "identifiers": [
+    {
+      "value": "2024-202454/AC/r967",
```

Comment:
This is not the correct format for Authority LCCNs.
For CONCEPT resources, let us use the following LCCNs
sh1234557890
sh 0123456789 (note that there is a space after "sh". This should get removed by the DefaultLccnNormalizer)